### PR TITLE
create house of dapps

### PR DESCRIPTION
**Pull request by metaverde**
Tidied up and added links to House of dApps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "build:serve": "docusaurus build && serve build/"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.64",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.64",
+    "@docusaurus/core": "^2.0.0-beta.ff31de0ff",
```
Comment (luxumbra) [inline package.json:13]: Hey @metaverde. Thanks for your first PR! 🥳 
This update to docusaurus is breaking the build. One file `docs/how-does-it-work/xp.md` has an `image` property in the frontmatter that seems to be invalid in the beta version of Docusaurus. If we revert Docusaurus to the previous version, the build succeeds. :) Was there a reason for the bump in versions? Going forwards, we will be migrating to beta.0 as part of  a wiki overhaul that @jonathanprozzi has started working on. 

It is good practice to run `yarn build` on your local build to check your updates before you push & make a PR. That way you know your work will deploy without issue. This is especially important if you're upgrading any of the existing packages for a project. 😄

File: package.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
     "react-rnd": "^10.2.4",
     "react-tooltip": "^4.2.10",
     "simple-oauth2": "^2.5.2",
-    "styled-components": "^5.2.1"
+    "styled-components": "^5.2.1",
+    "yarn": "^1.22.10"
```
Comment (luxumbra) [inline package.json:26]: Do we need to add yarn to project dependencies?

**Review by luxumbra — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (chair28980) [reply package.json:13]: The version bump was likely due to us walking through installing node/yarn/npm through the push process. We'll get it fixed and whipped into shape 👍

**Comment by luxumbra** [comment]
Closing this as the work was submitted in #259

**Pull request closed without merging**